### ISSUE TITLE
Crash after resizing canvas

### ISSUE THREAD

**Issue by Equbuxu** [PixiEditor/PixiEditor#322]
1. Draw a line across a new 64x64 canvas
1. Resize canvas to 32x32
1. Select an area
1. Try to move it

**Comment by Kwexy** [comment]
BitmapUtils.cs throws an exception if any of the layers extend beyond the canvas borders. I'm not sure what this exception is for, but commenting this out fixes the issue. Are layers not supposed to extend beyond canvas borders?

**Comment by Equbuxu** [comment]
> BitmapUtils.cs throws an exception if any of the layers extend beyond the canvas borders. I'm not sure what this exception is for, but commenting this out fixes the issue. Are layers not supposed to extend beyond canvas borders?

Correct, the layers must not extend beyond canvas borders, some operations rely on that. I'm working on this issue in #326
